packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -323,10 +304,6 @@ export default class Meetings extends WebexPlugin {
       this.webex.internal.mercury.on(ONLINE, () => {
         this.syncMeetings();
       });
-
-      this.webex.internal.mercury.on(OFFLINE, () => {
```

Comment:
This was a customer requested feature. Emitting the offline event at the top level of the meetings plugin doesn't interfere with our work. Please re-add.